frontend/.env.test

```diff
@@ -1,4 +1,5 @@
 NEXT_PUBLIC_ALGOLIA_APP_ID=XQEP3AD9ZT
 NEXT_PUBLIC_IFIXIT_ORIGIN=https://www.cominor.com
 NEXT_PUBLIC_STRAPI_ORIGIN=https://strapi.cominor.com
-NEXT_PUBLIC_SHOPIFY_STOREFRONT_VERSION=2021-07
\ No newline at end of file
+NEXT_PUBLIC_SHOPIFY_STOREFRONT_VERSION=2021-07
```

Comment:
Not this issue but should this be `2022-07` now like it is in `.env.production` and `.env.development`.

Reply:
Let's try it!

```suggestion
NEXT_PUBLIC_SHOPIFY_STOREFRONT_VERSION=2022-07
```